doc/arch/big-picture.md

```diff
@@ -190,12 +190,12 @@ So a simple event handle could just assoc what we need into the database.
   (assoc-in tx-env [:db ident :completed?] checked))
 ```
 
-Once the event completes the transaction sees that only `ident` was updated. It'll then check which queries used that `ident` earlier when rendering and signal that they need to update.
+Once the event completes the transaction it sees that only `ident` was updated. It'll then check which queries used that `ident` when rendering and signal that they need to be updated.
```

Comment:
I guess this is missing a comma? The transaction sees that only ident was updated, not the event.

Reply:
I think I know what you mean. What about this:

> Once the event completes, the transaction sees that `ident` was updated. It'll then check which queries used that `ident`, when rendering, and signal that they need to be updated.

Reply:
Sounds good.